package/gluon-state/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-state
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/$(PKG_NAME)
+  TITLE:=Provides info about the routers state
+  DEPENDS:=+gluon-core +micrond
+endef
+
+define Package/$(PKG_NAME)/description
+  gluon-state executes checks in `/lib/gluon/state/check.d/` and provides
+  a flag file for each check in `/var/gluon/state` depending on the return code
+  of the check. A flag file is created (or "touched") if the corresponding check
+  exits cleanly and gets removed otherwise. If the flags are "touched", they
+  are only accessed, but not modified. In this way, the atime of a flag file
+  reflects when the last check was performed and the mtime reflects when
+  when the state was last changed.
+
+  This package provides the following checks:
+  - `has_default_gw6` - check whether the router has a default IPv6-route on br-client.
+  
+  The checks are executed once every minute (by micron.d).
+endef
+
+$(eval $(call BuildPackageGluon,$(PKG_NAME)))
```

Comment:
The common style of other Gluon packages is not to use `$(PKG_NAME)`.

Reply:
Why is that and if there is no strong reason why not change all to $(PKG_NAME) for convenience?

Reply:
The latter question looks valid to me, but is out of this PRs scope. It can be taken care of over there (#2286).
Resolved in 8a813a7cf25cf9078209323b8a0b7d663cc56f1c in order to match the current codebase.